docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/FilterRowsByColumn.cs

```diff
@@ -5,9 +5,9 @@
 namespace Microsoft.ML.Samples.Dynamic
 {
     /// <summary>
-    /// Sample class showing how to use <see cref="DataOperationsCatalog.FilterByColumn"/>.
+    /// Sample class showing how to use FilterRowsByColumn.
```

Comment:
I wonder if `ColumnValue` might be better than just `Column` in this context, but maybe not. Probably fine...